ajax/libs/videojs-flash/package.json

```diff
@@ -0,0 +1,25 @@
+{
+  "name": "videojs-flash",
+  "filename": "videojs-flash.min.js",
+  "description": "The official Flash tech package for Video.js.",
+  "keywords": [
+    "videojs",
+    "videojs-plugin",
+    "videojs-tech"
+  ],
+  "author": "Brightcove, Inc.",
+  "license": "Apache-2.0",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/videojs/videojs-flash.git"
+  },
+  "npmName": "videojs-flash",
+  "npmFileMap": [
+    {
+      "basePath": "dist",
+      "files": [
+        "**/!(*.cjs.js|*.es.js)"
```

Comment:
Let's use `**/!(*.cjs.*|*.es.*)` to prevent possible other back-end files.

Reply:
Sure, I have modified the commit as you recommended.